tests/pytorch/nightly/hf-lm.libsonnet

```diff
@@ -192,6 +192,7 @@ local utils = import 'templates/utils.libsonnet';
         pip install tensorboardX google-cloud-storage
         echo 'export PATH=~/.local/bin:$PATH' >> ~/.bash_profile
         echo 'export XLA_USE_BF16=1' >> ~/.bash_profile
+        pip install pathlib
```

Comment:
NB: Could you move this pip install up to the same line as the others? e.g. line 192 could be `pip install tensorboardX google-cloud-storage pathlib`

Reply:
Good idea! Thanks, Steven, updated